frontend/src/app/models/pipeline.ts

```diff
@@ -34,30 +36,31 @@ export class Pipeline {
   @Type(() => Param)
   params: Param[] = [];
 
-  blocked_running(): boolean {
+  public blocked_running(): boolean {
```

Comment:
By default, all members of a class are public (unless your'e declaring a non-readonly public parameter property in a constructor) so consider removing the public modifier.

Reply:
Just thought this was threading along suggestions in the styleguide but you can leave as is if you prefer.
https://google.github.io/styleguide/tsguide.html#visibility